app/models/tattoo.rb

```diff
@@ -1,4 +1,5 @@
 class Tattoo < ActiveRecord::Base
+  has_many :reviews
```

Comment:
I want to see a test of this association reflected in the `Tattoo` unit test! https://learn.launchacademy.com/questions/314